packages/target-tauri/src-tauri/src/help_window.rs

```diff
@@ -17,8 +17,8 @@ impl serde::Serialize for Error {
 }
 
 #[tauri::command]
-pub(crate) fn open_help_window(
-    app: tauri::AppHandle,
+pub(crate) fn open_help_window<A: Runtime>(
+    app: tauri::AppHandle<A>,
```

Comment:
why is this needed?

Reply:
To make typechecker happy. I call this from the menu where Apphandle is more abstract